src/EmbeddedVsix/EmbeddedVsix.csproj

```diff
@@ -102,6 +129,9 @@
     <!-- Specifies the target inputs and outputs so MSBuild can skip fetching the files if the analyzer version has not changed. -->
     <AdditionalFilesInput Include="..\EmbeddedSonarAnalyzer.props" />
     <AdditionalFilesOutput Include="$(OutputPath)$(AssemblyName).vsix" />
+    
+    <!-- Prevent the vsix from being cleaned during incremental builds -->
```

Comment:
Shouldn't this property-set have been deleted? (now done in line 17)

Reply:
You're right, it's a conflict, I'll fix and force push